pkg/cmd/roachtest/tests/asyncpg.go

```diff
@@ -92,17 +87,34 @@ func registerAsyncpg(r registry.Registry) {
 			t.Fatal(err)
 		}
 
-		const cockroachPort = 26257
-		if err := repeatRunE(
-			ctx,
-			t,
-			c,
-			node,
-			"run setup.py test for asyncpg",
-			fmt.Sprintf("cd /mnt/data1/asyncpg && PGPORT=%d python3 setup.py test", cockroachPort),
+		blocklistName, expectedFailureList, ignoredlistName, ignoredlist := asyncpgBlocklists.getLists(version)
+		if expectedFailureList == nil {
+			t.Fatalf("No asyncpg blocklist defined for cockroach version %s", version)
+		}
+		if ignoredlist == nil {
+			t.Fatalf("No asyncpg ignorelist defined for cockroach version %s", version)
+		}
+		t.L().Printf("Running cockroach version %s, using blocklist %s, using ignorelist %s",
+			version, blocklistName, ignoredlistName)
+
+		t.Status("Running asyncpg tests ")
+		rawResults, _ := c.RunWithBuffer(
```

Comment:
should we log the error from this anyway?

Reply:
Not sure if I understand it correctly -- the error is already logged in one of the `cd.log` under `artifacts/asyncpg/run_1/`. [Here's an example](https://gist.github.com/ZhouXing19/82211c66aa2555df3af4f84103f79b6b).

Reply:
running the command can fail for other reasons, so i'd prefer if something happens we still have a record.

```
rawResults, err := c.RunWithBuffer(...)
if err != nil {
  t.Logf("error during asyncpg run (may be ok): %v\n", err)
}
```